core/domain/taskqueue_services.py

```diff
@@ -33,7 +33,6 @@
 platform_taskqueue_services = models.Registry.import_taskqueue_services()
 
 
-# NOTE: The following constants should match the queue names in queue.yaml.
```

Comment:
@Nik-09 I think you should mention that these need to be in sync with the queue names on the Cloud Tasks UI?

Reply:
Done